SoftLayer/CLI/email/detail.py

```diff
@@ -0,0 +1,72 @@
+"""Get details for an image."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+from SoftLayer.managers.account import AccountManager
+from SoftLayer.managers.email import EmailManager
+from SoftLayer import utils
+
+
+@click.command()
+@environment.pass_env
+def cli(env):
+    """"""
+    manager = AccountManager(env.client)
+    email_manager = EmailManager(env.client)
+    result = manager.get_Network_Message_Delivery_Accounts()
+
+    table = formatting.KeyValueTable(['name', 'value'])
+
+    table_information = formatting.KeyValueTable(['id', 'username', 'hostname', 'description', 'vendor'])
+    table_information.align['id'] = 'r'
+    table_information.align['username'] = 'l'
+
+    for email in result:
+        # print(email['id'])
+        table_information.add_row([email.get('id'), email.get('username'), email.get('emailAddress'),
+                                   utils.lookup(email, 'type', 'description'),
+                                   utils.lookup(email, 'vendor', 'keyName')])
+
+        overview_table = _build_overview_table(email_manager.get_AccountOverview(email.get('id')))
+        statistics = email_manager.get_statistics(email.get('id'),
+                                                  ["requests", "delivered", "opens", "clicks", "bounds"],
+                                                  True, True, True, 6)
```

Comment:
If these data are by default (**["requests", "delivered", "opens", "clicks", "bounds"], True, True, True, 6**) you do not need to send them as parameters, use them direct in the method. Send only the email id:

`statistics = email_manager.get_statistics(email.get('id'))`